requirements_test.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements_test_min.txt
+-r requirements_test_pre_commit.txt
```

Comment:
You mentioned here https://github.com/PyCQA/astroid/commit/664555ac1f26113c78397297cbf340fb64736a24 that this caused an error. I tried to setup a new venv and everything worked out fine. They seem to be resolved as a whole.
```bash
python -m venv venv
source venv/bin/activate
pip install -r requirements_test.txt
```